proxy-node-shared/src/main/java/uk/gov/ida/notification/contracts/verifyserviceprovider/Attribute.java

```diff
@@ -54,13 +55,19 @@ public DateTime getTo() {
         return to;
     }
 
-
-    public boolean isValid() {
-        return this.verified && this.isCurrent() && this.value != null;
-    }
-
     public boolean isCurrent() {
-        return (this.from != null && this.from.isBeforeNow()) &&
+        return (this.from == null || this.from.isBeforeNow()) &&
```

Comment:
This fixes the second A/C in https://govukverify.atlassian.net/browse/EID-1750 so we can update that ticket once this lands 👍